name_resolution/name_resolver.rb

```diff
@@ -0,0 +1,21 @@
+require 'resolv'
+
+class NameResolver < Scout::Plugin
+  OPTIONS=<<-EOS
+    nameserver:
+      default: 8.8.8.8
+    resolve_address:
+      default: google.com
+  EOS
+
+  def build_report
+    resolver = Resolv::DNS.new(:nameserver => [option(:nameserver)])
+
+    begin
+      result = resolver.getaddress(option(:resolve_address))
+    rescue Resolv::ResolvError => err
+      alert('Failed to resolve', err)
```

Comment:
Will this systematically trigger an alert if the namespace can't resolve once? Is this something we expect to happen rarely enough that we're happy getting an alert each time? Would we have more flexibility with a 0-1 metric to graph/set triggers over time?